src/main/java/core/basesyntax/Launcher.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+/*
+    + Read an input file, handle exception, store data in storage;
+    + Process data by specified storage keys;
+    + Check lines for consistent data "[bspr],\w++,\d++";
+    + Form a String report from calculated data in storage;
+    + Write the report in "./res/OutGoingReports/Report.csv", make sure to overwrite it;
+    + Perform additional checks for input/output files, negative values, etc.
+*/
+
+import logic.MainLogic;
+import logic.MainLogicImpl;
+
+public class Launcher {
```

Comment:
this is more of a ReadMe.md or a Javadoc type of information but we will not focus on that now